docs/testrunner-toolkit/configuration/espresso.md

```diff
@@ -291,3 +294,36 @@ __Example__:
 ```yaml
   annotation: com.android.buzz.MyAnnotation
 ```
+
+### `numShards`
+
+__Description__: `numShards` sets the number of separate shards to create. This option works for both emulators and RDC. For more shard tests, click [here](https://developer.android.com/training/testing/junit-runner#sharding-tests) to read more.
+
+__Type__: *int*
+
+__Example__:
+```yaml
+  numShards: 4
+```
+
+### `shardIndex`
+
+__Description__: `shardIndex` specifies which shard to run. This option works for both emulators and RDC. For more shard tests, click [here](https://developer.android.com/training/testing/junit-runner#sharding-tests) to read more.
```

Comment:
Do we need to mention that is it both supported by emulator and RDC ?
Since it's working everywhere, may be we should remove `This option works for both emulators and RDC.`

Reply:
sure.